junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvArgumentsProvider.java

```diff
@@ -47,26 +50,40 @@ public void accept(CsvSource annotation) {
 		settings.getFormat().setQuoteEscape('\'');
 		settings.setEmptyValue(this.annotation.emptyValue());
 		settings.setAutoConfigurationEnabled(false);
+
 		CsvParser csvParser = new CsvParser(settings);
 		AtomicLong index = new AtomicLong(0);
 
+		DefaultConversionProcessor processor = getConversionProcessor();
```

Comment:
If this.annotation.nullSymbols() is equals to default value, the conversionProcessor could be skiped.

Reply:
Hi @Ciruman,

have a look at [my other comment](https://github.com/junit-team/junit5/pull/1883#discussion_r286417700) for my explanation.